src/GraphQLParser.Tests/ParserTests.cs

```diff
@@ -773,12 +773,63 @@ public void Should_Parse_Unions(string text)
         }
 
         [Theory]
-        [InlineData("type Query", ASTNodeKind.ObjectTypeDefinition)]
-        [InlineData("extend type Query", ASTNodeKind.TypeExtensionDefinition)]
```

Comment:
`extend type Query` actually is invalid query :) , see `Should_Throw_Extensions`